dependencies.yaml

```
@@ -86,6 +89,15 @@ files:
     includes:
       - develop
       - py_version
+  clang_tidy:
+    output: none
+    includes:
+      - build_all
+      - build_base
+      - clang_tidy
+      - cuda
```

If this includes cuda, it should include cuda_version too.
```suggestion
      - cuda
      - cuda_version
```